apps/da/da-cos-api/da-cos-api.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: da-cos-api
+spec:
+  releaseName: da-cos-api
+  interval: 1m
+  values:
+    java:
+      image: hmctspublic.azurecr.io/da/cos-api:latest* #{"$imagepolicy": "flux-system:adoption-cos-api"}
```

Comment:
image policy looks wrong? (adoption)

Reply:
updated to - image: hmctspublic.azurecr.io/da/cos-api:latest  #{"$imagepolicy": "flux-system:da-cos-api"}